source/base/quadrature.cc

```diff
@@ -91,8 +118,11 @@ template <int dim>
 Quadrature<dim>::Quadrature (const Point<dim> &point)
   :
   quadrature_points(std::vector<Point<dim> > (1, point)),
-  weights(std::vector<double> (1, 1.))
-{}
+  weights(std::vector<double> (1, 1.)),
+  is_tensor_product_flag (dim==1)
```

Comment:
But we a single point and arbitrary combinations of 1d formulas the formula is always a tensor product, isn't it?